fuel/src/main/kotlin/com/github/kittinunf/fuel/core/Request.kt

```diff
@@ -146,7 +146,7 @@ class Request(
     fun body(body: String, charset: Charset = Charsets.UTF_8): Request = body(body.toByteArray(charset))
 
     fun jsonBody(body: String, charset: Charset = Charsets.UTF_8): Request {
-        header("content-type" to "application/json")
+        headers["Content-Type"] = "application/json"
         return body(body, charset)
     }
```

Comment:
should we amend the test for this `RequestTest.kt` line 301, currently the test passes and if it is a bug the test should also be changed to reflect the desired behaviour

Reply:
Agree with that. So we have test that covered the behavior.